examples/amp-story/performance/prestyle.html

```diff
@@ -0,0 +1,112 @@
+<!-- Imports the story prestyle CSS so it can be laid out before the JS loads -->
+<!doctype html>
+<html ⚡ lang="en">
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="lcp.html">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <title>Boilerplate test</title>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-story" src ="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+  <!-- Fonts -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="/css/amp-story-prestyle.css">
```

Comment:
This should be `https://cdn.ampproject.org/v0/amp-story-1.0.css` and get automatically rewritten. Can you look into what that involves and/or create a ticket?